assets/js/modules/search-console/components/dashboard/DashboardPopularKeywordsWidget.js

```diff
@@ -70,6 +73,46 @@ function DashboardPopularKeywordsWidget( { Widget, WidgetReportZero, WidgetRepor
 		return <WidgetReportZero moduleSlug="search-console" />;
 	}
 
+	const tableColumns = [
+		{
+			title: __( 'Keyword', 'google-site-kit' ),
+			description: __( 'Most searched for keywords related to your content', 'google-site-kit' ),
+			primary: true,
+			field: 'keys.0',
+			Component: ( { fieldValue } ) => {
+				const searchAnalyticsURL = useSelect( ( select ) => {
+					const dates = select( CORE_USER ).getDateRangeDates( { offsetDays: DATE_RANGE_OFFSET } );
+					const entityURL = select( CORE_SITE ).getCurrentEntityURL();
+					return select( MODULES_SEARCH_CONSOLE ).getServiceReportURL( {
+						...generateDateRangeArgs( dates ),
+						query: `!${ fieldValue }`,
+						page: entityURL ? `!${ entityURL }` : undefined,
+					} );
+				} );
+
+				return (
+					<Link
+						href={ searchAnalyticsURL }
+						external
+						inherit
+					>
+						{ fieldValue }
+					</Link>
+				);
+			},
+		},
+		{
+			title: __( 'Clicks', 'google-site-kit' ),
+			description: __( 'Number of times users clicked on your content in search results', 'google-site-kit' ),
+			Component: ( { row } ) => numFmt( row.clicks, { style: 'decimal' } ),
+		},
+		{
+			title: __( 'Impressions', 'google-site-kit' ),
+			description: __( 'Counted each time your content appears in search results', 'google-site-kit' ),
+			Component: ( { row } ) => numFmt( row.impressions, { style: 'decimal' } ),
+		},
+	];
```

Comment:
Same here: This is probably not an issue, but was just curious why you moved this block of code.

Reply:
Same here: to make sure translations are loaded by the time we render this component.